tf_keras/callbacks.py

```diff
@@ -629,9 +629,9 @@ class Callback:
     inference lifecycle.
 
     To create a custom callback, subclass `keras.callbacks.Callback` and
-    override the method associated with the stage of interest. See
-    https://www.tensorflow.org/guide/tf_keras/custom_callback for more
-    information.
+    override the method associated with the stage of interest. See the 
```

Comment:
@RenuPatelGoogle there is a trailing whitespace on this line, can you please remove it?

Thanks